src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/UpdateFee.tsx

```diff
@@ -63,7 +63,7 @@ const UpdateFee = () => {
     applicationStore.setIsLoading(true);
     Operator.getInstance().getOperator(operatorStore.processOperatorId).then(async (response: any) => {
       if (response) {
-        const operatorFee = formatNumberToUi(ssvStore.getFeeForYear(walletStore.fromWei(response.fee)));
+        const operatorFee = formatNumberToUi(ssvStore.getFeeForYear(walletStore.fromWei(response.fee))).replace(',', '');
```

Comment:
@axelrod-blox this looks like a dirty hack, can you get the fee in a more deterministic way instead of splitting by comma?

Reply:
done